app/assets/stylesheets/application.scss

```diff
@@ -57,3 +57,6 @@ div.review-image {
   margin-right: auto;
   margin-left: auto;
 }
+input#search.search {
+  background-color: #333;
```

Comment:
Color literals like `#333` should only be used in variable declarations; they should be referred to via variable everywhere else.